src/app/portlets/content-types/form/content-types-form.component.spec.ts

```diff
@@ -802,8 +732,4 @@ describe('ContentTypesFormComponent', () => {
             system: null
         });
     });
-
-    xit('should reset and collapse the form on ESC key', () => {
-        // TODO: waiting for Luis mock hotkeys.
-    });
```

Comment:
I think we need a new test to check when the edit button is clicked, Also I think we need a test to check the submitForm method is called when the button is clicked